docker/Dockerfile-opflex-build-base

```diff
@@ -5,13 +5,30 @@ RUN microdnf install --enablerepo codeready-builder-for-rhel-8-x86_64-rpms \
     libtool pkgconfig autoconf automake make cmake file python2-six \
     openssl-devel git gcc gcc-c++ boost-devel diffutils python2-devel \
     libnetfilter_conntrack-devel wget which curl-devel procps zlib-devel \
+    libmnl-devel vi \
     && microdnf clean all
+RUN git clone https://git.netfilter.org/libnftnl \
```

Comment:
Can we pull specific tags for libnftlnl and iptables instead of tracking master?

Reply:
ok. let me check the latest tags and resend.

Reply:
picked libnftnl-1.1.7 and iptables 1.8.5 thanks.